charts/prometheus-kafka-exporter/values.yaml

```
@@ -187,6 +187,14 @@ sasl:
     # or use an existing secret
     # secretName: <name of an existing secret>
 
+  # If enabled, the exporter will use AWS IAM authentication for SASL.
+  # Be aware that this requires the eks.amazonaws.com/role-arn annotation to be set on the ServiceAccount
+  # and most probably the tls.enabled to be set to true.
+  awsiam:
+    enabled: false
```

We may wish to add support for AWS region, too:
```suggestion
    enabled: false
    awsRegion: ""
```